refinery/config/settings/base.py

```diff
@@ -577,3 +578,5 @@ def get_setting(name, settings=local_settings):
 REFINERY_TUTORIAL_STEPS = refinery_tutorial_settings
 
 ANONYMOUS_USER_ID = -1
+
+TEST_RUNNER = "django.test.runner.DiscoverRunner"
```

Comment:
@hackdna Not sure, will have to do some reading. This was the docs suggestion to suppress the warnings that system check was throwing on test runs